packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -1,85 +1,431 @@
+ * Called with a [[GridCellParams]] object.
+ * @event
+ */
+export const GRID_CELL_DRAG_ENTER = 'cellDragEnter';
+
+/**
+ * Fired while an element or text selection is dragged over the cell.
+ * It's mapped to the `dragover` DOM event.
+ * Called with a [[GridCellParams]] object.
+ * @event
+ */
+export const GRID_CELL_DRAG_OVER = 'cellDragOver';
+
+/**
+ * Fired when the dragging of a cell ends. Called with a [[GridCellParams]] object.
+ * @event
+ */
+export const GRID_CELL_DRAG_END = 'cellDragEnd';
+
+/**
+ * Fired when the props of the edit input changes. Called with a [[GridEditCellPropsParams]] object.
+ * @event
+ */
 export const GRID_CELL_EDIT_PROPS_CHANGE = 'cellEditPropsChange';
+
+/**
+ * Fired when the props of the edit input are committed. Called with a [[GridEditCellPropsParams]] object.
+ * @event
+ */
 export const GRID_CELL_EDIT_PROPS_CHANGE_COMMITTED = 'cellEditPropsChangeCommitted';
+
+/**
+ * Fired when the value of a cell changes. Called with a [[GridEditCellValueParams]] object.
+ * @event
+ */
 export const GRID_CELL_VALUE_CHANGE = 'cellValueChange';
 
+/**
+ * Fired when the cell turns to edit mode. Called with a [[GridCellParams]] object.
+ * @event
+ */
 export const GRID_CELL_EDIT_ENTER = 'cellEditEnter';
+
+/**
+ * Fired when the cell turns back to view mode. Called with a [[GridCellParams]] object.
+ * @event
+ */
 export const GRID_CELL_EDIT_EXIT = 'cellEditExit';
+
+/**
+ * Fired when a [navigation key](/components/data-grid/accessibility#keyboard-navigation) is pressed in a cell.
+ * Called with a [[GridCellParams]] object.
+ * @event
+ */
 export const GRID_CELL_NAVIGATION_KEYDOWN = 'cellNavigationKeyDown';
-export const GRID_CELL_FOCUS = 'cellCellFocus';
-export const GRID_CELL_DRAG_START = 'cellDragStart';
-export const GRID_CELL_DRAG_ENTER = 'cellDragEnter';
-export const GRID_CELL_DRAG_OVER = 'cellDragOver';
-export const GRID_CELL_DRAG_END = 'cellDragEnd';
 
+/**
+ * Fired when a row is clicked. Called with a [[GridRowParams]] object.
+ * @event
+ */
 export const GRID_ROW_CLICK = 'rowClick';
+
+/**
+ * Fired when a row is double-clicked. Called with a [[GridRowParams]] object.
+ * @event
+ */
 export const GRID_ROW_DOUBLE_CLICK = 'rowDoubleClick';
+
+/**
+ * Fired when a `mouseover` event happens in a row.  Called with a [[GridRowParams]] object.
+ * @event
+ */
 export const GRID_ROW_OVER = 'rowOver';
+
+/**
+ * Fired when a `mouseout` event happens in a row.  Called with a [[GridRowParams]] object.
+ * @event
+ */
 export const GRID_ROW_OUT = 'rowOut';
+
+/**
+ * Fired when a `mouseenter` event happens in a row.  Called with a [[GridRowParams]] object.
+ * @event
+ */
 export const GRID_ROW_ENTER = 'rowEnter';
+
+/**
+ * Fired when a `mouseleave` event happens in a row.  Called with a [[GridRowParams]] object.
+ * @event
+ */
 export const GRID_ROW_LEAVE = 'rowLeave';
+
+/**
+ * @ignore - do not document.
+ */
 export const GRID_ROW_EDIT_MODEL_CHANGE = 'editRowModelChange';
```

Comment:
Why to publish this event if there's no row editing feature?

Reply:
This is the editing model that works for rows and cells.